client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-card/index.tsx

```diff
@@ -96,6 +100,22 @@ export default function HostingCard( {
 		return translate( 'USD' );
```

Comment:
It's not from this PR, but I'm curious about currency translations. As far as I know, ISO currency should not be translated.

Reply:
Thank you for raising this. As I'm not 100% sure about this I'd ask @jkguidaven and @yashwin.
I will deploy this PR in the meantime as it's doesn't update this logic.